File: Go/LED_Numbers.go
package main

import "fmt"

func main() {
	fmt.Print("Enter a number: ")
	var num string
	fmt.Scanf("%s", &num)
	leds := map[string][]string {
		"0" : []string{" _  ","| | ","|_| "},
		"1" : []string{"  ","| ","| "},
		"2" : []string{" _  "," _| ","|_  "},
		"3" : []string{"_  ","_| ","_| "},
		"4" : []string{"    ","|_| ","  | "},
		"5" : []string{" _  ","|_  "," _| "},
		"6" : []string{" _  ","|_  ","|_| "},
		"7" : []string{"_   "," |  "," |  "},
		"8" : []string{" _  ","|_| ","|_| "},
		"9" : []string{" _  ","|_| "," _| "},
	}
	for i := 0; i < 3; i++ {
		for j := 0; j < len(num); j++ {
			fmt.Print(leds[string(num[j])][i])
		}
		fmt.Print("\n")
	}
}

File: Dart/LEDNumbers.dart
import 'dart:io';
import 'dart:collection';

void main() {
	var leds = HashMap.from({'0': [" _  ","| | ","|_| "], 
		                     '1': ["  ","| ","| "],
		                     '2': [" _  "," _| ","|_  "],
		                     '3': ["_  ","_| ","_| "],
		                     '4': ["    ","|_| ","  | "],
		                     '5': [" _  ","|_  "," _| "],
		                     '6': [" _  ","|_  ","|_| "],
		                     '7': ["_   "," |  "," |  "],
		                     '8': [" _  ","|_| ","|_| "],
		                     '9': [" _  ","|_| "," _| "]});
	
	var line1 = '';
	var line2 = '';
	var line3 = '';
	stdout.write('Enter a number: ');
	String num = stdin.readLineSync() ?? ''; 
	List<String> nums = num.split('');
	
	for (var i = 0; i < nums.length; i++) {
		line1 += leds[nums[i]][0];
		line2 += leds[nums[i]][1];
		line3 += leds[nums[i]][2];
	}
	
	print("$line1");
	print("$line2");
	print("$line3");
}
